### DISCUSSION TITLE
html引入客户端如何操作？

### DISCUSSION THREAD

**Discussion by moksha007** [walinejs/waline#901] [Q&A] [answered]
主题：butterfly    最后这一步不知道如何操作。评论显示弹出：未初始化。导致评论无法生效。我没有操作最后一步，完全看不懂。请教大神，如何解决。

![image](https://user-images.githubusercontent.com/96851605/162347184-0bb563c8-51f5-4221-bbe0-5f9c920daeef.png)

**Comment by lizheming** [comment, marked as answer]
https://butterfly.js.org/posts/ceeb73f/#%E8%A9%95%E8%AB%96 butterfly 已经自带 waline 的支持，建议看下主题文档
